common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OAuth2TokenCache.java

```diff
@@ -139,12 +140,12 @@ public abstract AccountRecord getAccountWithLocalAccountId(final String environm
      * @param clientId      The clientId of this current app.
      * @param homeAccountId The homeAccountId of the Account targeted for deletion.
      * @param realm         The tenant id of the targeted Account (if applicable).
-     * @return True, if the Account was deleted. False otherwise.
+     * @return The {@link AccountRecordDeletionResult} containing the removed AccountRecords.
      */
-    public abstract boolean removeAccount(final String environment,
-                                          final String clientId,
-                                          final String homeAccountId,
-                                          final String realm
+    public abstract AccountRecordDeletionResult removeAccount(final String environment,
```

Comment:
I'd consider AccountDeletionRecord instead of Result... since I don't believe that's it's possible for the deletion of a specific account to fail.

Reply:
👍